algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/workload.py

```diff
@@ -47,6 +49,7 @@ class LibriSpeechConformerWorkload(workload.BaseLibrispeechWorkload):
 
   def init_model_fn(self, rng: spec.RandomState) -> spec.ModelInitState:
     torch.random.manual_seed(rng[0])
+    torch.backends.cudnn.benchmark = False
```

Comment:
have you confirmed that this is still necessary to prevent an OOM? if so can you add a comment saying why we need this?

Reply:
Yes, added a comment!